spring-cloud-gcp-docs/src/main/asciidoc/datastore.adoc

```diff
@@ -382,7 +382,36 @@ public class MyApplication {
 
 ==== Query methods by convention
 
-This feature is not supported yet.
+[source, java]
+----
+public interface TradeRepository extends CrudRepository<Trade, String[]> {
+    List<Trader> findByAction(String action);
+
+	int countByAction(String action);
+
+  boolean existsByAction(String action);
```

Comment:
formatting (tab vs spaces)

Reply:
@meltsufin changes made and issues created. PTAL